x-pack/plugins/ml/public/application/routing/routes/data_frame_analytics/analytics_jobs_list.tsx

```diff
@@ -20,6 +20,7 @@ export const analyticsJobsListRouteFactory = (
   navigateToPath: NavigateToPath,
   basePath: string
 ): MlRoute => ({
+  id: 'data_frame_analytics',
```

Comment:
Looks like we could use the `ML_PAGES` locator constants we have here https://github.com/elastic/kibana/blob/3cca7f06305e2a322f1b92adabed2c7338b88631/x-pack/plugins/ml/common/constants/locator.ts

Reply:
`ML_PAGES` contains pathname, `id` isn't always the same string as a path.

Reply:
ah yes. maybe we should have a constant for these ids (as a future improvement)